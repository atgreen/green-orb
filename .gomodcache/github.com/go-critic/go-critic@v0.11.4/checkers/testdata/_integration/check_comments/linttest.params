check -enable=commentedOutImport,codegenComment ./... | linttest.golden
