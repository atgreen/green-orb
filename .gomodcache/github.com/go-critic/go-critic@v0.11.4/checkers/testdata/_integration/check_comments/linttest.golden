exit status 1
./autogen1.go:1:1: codegenComment: comment should match `Code generated .* DO NOT EDIT.` regexp
./autogen2.go:1:1: codegenComment: comment should match `Code generated .* DO NOT EDIT.` regexp
./autogen3.go:1:1: codegenComment: comment should match `Code generated .* DO NOT EDIT.` regexp
./autogen4.go:1:1: codegenComment: comment should match `Code generated .* DO NOT EDIT.` regexp
./foo.go:5:2: commentedOutImport: remove commented-out "os" import
./foo.go:6:2: commentedOutImport: remove commented-out "fmt" import
./foo.go:6:2: commentedOutImport: remove commented-out "strconv" import
./foo.go:12:2: commentedOutImport: remove commented-out "foo/bar" import
./foo.go:13:2: commentedOutImport: remove commented-out "foo/bar/baz" import